mantle/kola/harness.go

```diff
@@ -194,6 +194,9 @@ const (
 	// kolaExtBinDataEnv is an environment variable pointing to the above
 	kolaExtBinDataEnv = "KOLA_EXT_DATA"
 
+	// kolaContainerDataEnv includes the path to the ostree base container image in oci-archive format.
+	kolaContainerDataEnv = "KOLA_OSTREE_OCIARCHIVE"
```

Comment:
The other variables follow the pattern `kolaExt-` which indicates that this constant is used by external tests. Maybe we can rename this to `kolaExtContainerDataEnv`?

Reply:
Fixed!  Good suggestion.